268_missing_number.py

```diff
@@ -0,0 +1,15 @@
+from typing import List
+
+
+class Solution:
+    def missingNumber(self, nums: List[int]) -> int:
```

Comment:
This solution is wrong, so correct it to be accepted.

Reply:
I tried the different way. Please check